data-serving/data-service/test/model/data/demographics.full.json

```diff
@@ -3,7 +3,7 @@
         "start": 20,
         "end": 29
     },
-    "sex": "Female",
+    "gender": "Female",
```

Comment:
Now that this is gender, rather than sex, should some test data cover values aside from "Male"/"Female"?

Reply:
Good call, changed up 3 tests (one for Non-binary/Third gender and one for Other)